gemrb/GUIScripts/GUIOPT.py

```diff
@@ -111,15 +111,22 @@ def InitOptionsWindow (Window):
 ToggleOptionsWindow = GUICommonWindows.CreateTopWinLoader(2, "GUIOPT", GUICommonWindows.ToggleWindow, InitOptionsWindow, None, GUICommonWindows.DefaultWinPos, True)
 OpenOptionsWindow = GUICommonWindows.CreateTopWinLoader(2, "GUIOPT", GUICommonWindows.OpenWindowOnce, InitOptionsWindow, None, GUICommonWindows.DefaultWinPos, True)
 
-def TrySavingConfiguration():
-	if not GemRB.SaveConfig():
-		print("ARGH, could not write config to disk!!")
-
 ###################################################
 
-def CloseVideoOptionsWindow ():
-	GemRB.GetView("SUB_WIN", 0).Close()
-	TrySavingConfiguration()
+def SaveSettings(Window, settings):
+	# volume sliders are multiplers, so apply thay before commition
```

Comment:
This comment can be improved. :)

I'm sceptical this will work, but let's see once the job is done.